commit e8cea8d70fed7ad5e14d8b3e871ebf0ea0bf53b0
Author: Steve Holme <steve_holme@hotmail.com>
Date:   Wed Nov 5 12:42:35 2014 +0000

    url.c: Fixed compilation warning
    
    conversion from 'curl_off_t' to 'size_t', possible loss of data

diff --git a/lib/url.c b/lib/url.c
index 40d4fed59..3a877fc87 100644
--- a/lib/url.c
+++ b/lib/url.c
@@ -368,7 +368,8 @@ CURLcode Curl_dupset(struct SessionHandle *dst, struct SessionHandle *src)
   i = STRING_COPYPOSTFIELDS;
   if(src->set.postfieldsize && src->set.str[i]) {
     /* postfieldsize is curl_off_t, Curl_memdup() takes a size_t ... */
-    dst->set.str[i] = Curl_memdup(src->set.str[i], src->set.postfieldsize);
+    dst->set.str[i] = Curl_memdup(src->set.str[i],
+                                  curlx_sotouz(src->set.postfieldsize));
     if(!dst->set.str[i])
       return CURLE_OUT_OF_MEMORY;
     /* point to the new copy */
